tests/unit/test_loader.py

```diff
@@ -97,10 +96,9 @@ def testDumpWholeConfig(self):
     a b
   </block>
 </a>
-a b
+a "b b"
 <a>
   <a block>
-    c "d d"
     b 三
```

Comment:
We had to remove this because the ordering between `c "d d"` and `b 三` was different in Python 3.3/3.4 from other versions of Python. We're still testing the "quoted value" in the above `a "b b"` statement.

Reply:
If ordering is still stable within any single version, may be explicit branching (separate test per distinct Python version) might be better than loosing the coverage a bit?

Reply:
It seems a bit brittle to test against a Python minor version, so I test against both orderings here instead.